README.md

```diff
@@ -1,12 +1,72 @@
-## Project Board
-https://app.zenhub.com/workspaces/commit-zero-5da8decc7046a60001c6db44/board?filterLogic=any&repos=203630543,247773730,257676371,258369081
-
 # Zero Deployable Frontend
 
-This is a [zero](https://github.com/commitdev/zero) module which sets up a
-service which can be deployed to the environment set up with [zero-aws-eks-stack](https://github.com/commitdev/zero-aws-eks-stack).
-It also contains a simple CircleCI pipeline which defines how to build and deploy the assets.
+This is a [Zero] module which sets up a
+service which can be deployed to the environment set up with [zero-aws-eks-stack][zero-infra].
+
+The `/templates` folder is meant to be filled in via [Zero][zero] and results in Simple React application. It also contains a simple CircleCI pipeline which defines how to build and deploy the service.
+
+This repository is framework/business-logic agnostic; mainly showcasing some universal best practices:
+
+- Creating an optimized build of your frontend application.
+
+- Out of the box CI/CD flow CircleCi.
+  - testing
+  - uploading build to an AWS S3 bucket.
```

Comment:
- building your web-app as a static distribution in /dist folder 
- syncs the built bundle to AWS s3
- invalidates Cloudfront cached version